crates/uv-toolchain/src/discovery.rs

```diff
@@ -19,6 +19,7 @@ use crate::virtualenv::{
 use crate::{Interpreter, PythonVersion};
 use std::borrow::Cow;
 
+use same_file::is_same_file;
```

Comment:
Import sections here are all over the place lol.

Reply:
Sorry that's probably my fault I use the editors "add import" feature a lot and things can get out of hand.